CHANGELOG.md

```diff
@@ -25,7 +25,7 @@
 * Update `generic_field_check` to have option to show percentage of sites that fail checks. [(#284)](https://github.com/broadinstitute/gnomad_methods/pull/284)
 * Added VEPed context HT to resource files and included support for versioning [(#282)](https://github.com/broadinstitute/gnomad_methods/pull/282)
 * Modified `vep_or_lookup_vep` to support the use of different VEP versions [(#282)](https://github.com/broadinstitute/gnomad_methods/pull/282)
-
+* Removed `rep_on_read`; this function is no longer necessary, as MatrixTables/Tables can be repartitioned on read with this [hail update](https://github.com/hail-is/hail/pull/9887) [(#283)](https://github.com/broadinstitute/gnomad_methods/pull/283)
```

Comment:
Maybe change "can be repartitioned on read with this" to "can be repartitioned on read with `_n_partitions`"